node/bft/src/sync/mod.rs

```diff
@@ -487,13 +487,14 @@ impl<N: Network> Sync<N> {
                 _ => bail!("Received a block with an unexpected authority type."),
             };
             let commit_round = leader_certificate.round();
-            let certificate_round = commit_round.saturating_add(1);
+            let certificate_round = commit_round.checked_add(1).unwrap();
```

Comment:
Do not include `unwraps` in non-testing context. 

```suggestion
            let certificate_round = commit_round.checked_add(1).ok_or_else(|| anyhow!("Integer overflow on round number"))?;
```